config/config.ts

```diff
@@ -113,6 +113,12 @@ export default {
               component: './Admin',
               authority: ['admin'],
             },
+            {
+              name: 'list.table-list',
+              icon: 'table',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'icon' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4665582)